internal/services/postgres/postgresql_flexible_server_resource.go

```diff
@@ -469,6 +470,30 @@ func resourcePostgresqlFlexibleServer() *pluginsdk.Resource {
 		},
 		),
 	}
+
+	if !features.FivePointOh() {
```

Comment:
I may be reading this wrong but to me, this looks the same in v4 as is it does in v5? In case I'm missing something, can you confirm that there is a difference between v4 and v5 that warrants putting this in a 5.0 block?